README.md

```diff
@@ -181,20 +250,29 @@ Table Data Location Examples
 | Grafana       | /dashboard-data/grafana           |
 | Mqtt          | /dashboard-data/ mqtt/credentials |
 | Nginx         | /dashboard-data/docker-nginx/authdata|
-| Certificates  | /dashboard-data/docker-nginx/letsencrypt
+| Let's Encrypt certificates  | /dashboard-data/docker-nginx/letsencrypt |
+| DNCserver     | /dashboard-data/apiserver/dncserver     |
+| DNCgiplugin   | /dashboard-data/apiserver/dncgiplugin`  |
+| DNCstdplugin  | /dashboard-data/apiserver/dncstdplugin` |
+| Mongodb       | /dashboard-data/mongodb/mongodb_data`   |
+| Expo          | /dashboard-data/expo/dncui              |
 
 ## Reuse and removal of data files
 
 Since data files on the host are not removed between runs, as long as the files are not removed between runs, the data will be preserved.
 
-Sometimes this is inconvenient, and it is necessary to remove some or all of the data. For a variety of reasons, the data files and directories are created owned by root, so the `sudo` command must be used to remove the data files. Here's an example of how to do it:
+Sometimes this is inconvenient, and it is necessary to remove some or all of the data. For a variety of reasons, the data files and directories are created owned by root, so the `sudo` command must be used to remove the data files. Here's an example of how to do it:
```

Comment:
Again, what changed? The highlighted spaces either were blanks before and are not now, or they are blanks now and were not before. Please use `xd` to get a hex dump to see what's going on.

Reply:
Yes. In visual studio code, it shows as small box(square) around the change.